src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,314 @@
+define only `_success` - the action to be performed is `product-to-body`  which we'll also define.
+
+```hocon
+actions {
+  fetch-product-with-cache {
+    factory = in-memory-cache
+    config {
+      cache {
+        maximumSize = 1000
+        ttl = 5000
+      }
+      cacheKey = "product-{param.id}"
+      payloadKey = fetch-product
+    }
+    doAction = fetch-product
+  }
+
+  product-to-body {
+    factory = payload-to-body
+    config {
+      key = fetch-product
+    }
+  }
+
+  fetch-product {
+    factory = http
+    config {
+      endpointOptions {
+        path = /product/id
+        domain = webapi
+        port = 8080
+        allowedRequestHeaders = ["Content-Type"]
+      }
+    }
+  }
+}
+```
+Above you can see all necessary actions definitions. As mentioned before, we've created `fetch-product-with-cache`. The purpose 
+of this action is to add cache functionality. Now you can see how easily we can define custom behaviours for our actions. 
+We've just assigned to it `in-memory-cache` action factory, moreover this action also wraps other - in this case it's 
+`fetch-product` - it's using simple http factory which will call desired endpoint - in our case `/product/id` (we'll define it later).
+Finally we can focus on `product-to-body` - behaviour of this action is really simple - it's just adding previously fetched data
+to response body.
+
+Last thing to do here is to combine this two chunks of code into one file, let's call it `fragments-handler.conf ` and put it
+inside `knotx/conf/routes/handlers/` directory.
+
+## Why we need caching?
+We've just added caching behaviour to action but you may wonder why we did it? Why we're using cache? The general purpose of
+using caching is faster data retrieval - when user call the same endpoint for few times, we don't have to fetch data
+from server for every request - we can cache it and reuse response (if not changed) in the future - it affects in quicker
+response time, what is noticeable by the user. On the other hand, from server side it reduces latency and traffic. Requests
+are satisfied in a shorter time with reused representations.
+
+You may also wonder if we can replace this cache with another one, for example with Redis - yes you can do it very simple!
+The only thing needed here is to create proper action factory which will handle desired cache and properly link it to action
+in config file (the same way is we did above).
+
+## Operations configuration
+Now we have to configure `knotx/conf/routes/operations.conf`. In our config we should have define two operations:
+- `healthcheck-operation`
+- `product-api-caching-proxy-operation`
+
+This operations are connected with `openapi.yaml` which we'll create next. This file specifies which handlers should be invoked 
+to satisfy request. Now we'll focus on operation called `product-api-caching-proxy-operation` (it will be connected to `openapi.yaml`).
```

Comment:
```suggestion
to process the request. Now we'll focus on operation called `product-api-caching-proxy-operation` (it will be connected to `openapi.yaml`).
```

Reply:
fixed